A44-xds-retry.md

```diff
@@ -0,0 +1,135 @@
+
+There are some key differences between gRPC’s and Envoy’s retry policy:
+
+- Envoy has the notion of [per_try_timeout](https://github.com/envoyproxy/envoy/blob/a67d8e588532fc2e3a7d89ee6aa469931c6419be/api/envoy/config/route/v3/route_components.proto#L1269) which current gRPC retry policy doesn’t have. In gRPC retry, when the deadline is exceeded for a request attempt, the RPC fails as a whole and there will be no retry possible after that. Whereas Envoy can set a timeout (earlier than the overall deadline) for each attempt, and in addition, all retries are contained within the overall request timeout.
+- Envoy’s [HedgePolicy](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#envoy-v3-api-msg-config-route-v3-hedgepolicy) is fundamentally different from gRPC’s HedgingPolicy. Envoy can have RetryPolicy and HedgePolicy specified together. In fact, Envoy’s HedgePolicy currently is only based on RetryPolicy’s per_try_timeout: a request attempt can only be hedged on per_try_timeout. If a request attempt fails prior to per_try_timeout, it might be retried after an exponential backoff, but will not be hedged after a hedging delay. On the contrary, gRPC can specify only one of the retry and hedging policy at a time for each method, but not both.
+- gRPC uses retry throttling to prevent server overload, whereas Envoy uses retry budgets or maximum active retries circuit breakers for the same purpose.
+  - Throttler config is a subconfig of service config, whereas retry circuit breaker is part of CDS update or cluster_impl lb config.
+  - Throttler is shared per client channel, whereas retry circuit breaker is shared per upstream cluster.
+
+### Related Proposals
+
+- [A6: gRPC Retry Design](https://github.com/grpc/proposal/blob/master/A6-client-retries.md)
+- [A31: gRPC xDS Timeout Support and Config Selector Design](https://github.com/grpc/proposal/blob/master/A31-xds-timeout-support-and-config-selector.md)
+
+## Proposal
+
+### Features to support
+
+Due to the differences between gRPC and Envoy, incomplete implementation status of the original [retry gRFC](https://github.com/grpc/proposal/blob/master/A6-client-retries.md) and unresolved [compatibility issues](https://github.com/envoyproxy/envoy/issues/17412) with dynamic cluster selection, we propose initially to support only some of the retry features in Envoy.
```

Comment:
I think this can just say "gRPC will support the following xDS retry features."  The details of why we aren't going to support individual features can be covered below.